code/script.js

```diff
@@ -32,16 +37,177 @@ const showMessage = (message, sender) => {
 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
 
-// Set up your eventlisteners here
+  showMessage(`Hello Skier, what's your name?`, 'bot')
+}
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
 // This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+setTimeout(greeting, 900)
+
+const reply = (input) => {
+  showMessage(input, 'user')
```

Comment:
Add a function, for example; userInput.value = '', to clear name input to an empty string.